src/System.Linq.Expressions/tests/CompilerTests.cs

```diff
@@ -115,6 +115,29 @@ public static void EmitConstantsToIL_LiftedToClosure()
             VerifyEmitConstantsToIL((DateTime?)DateTime.Now, 1);
         }
 
+        [Fact]
+        public static void VerifyIL_Simple()
+        {
+            Expression<Func<int>> f = () => Math.Abs(42);
+            VerifyIL(f, @"
+IL_0000: ldc.i4.s   42
+IL_0002: call       Int32 Abs(Int32)/System.Math
+IL_0007: ret        
+");
```

Comment:
Nit: if it doesn't already, could we change the verification code to not care about whitespace differences?  Then this text could be indented to make the test look a bit nicer.